pygeoapi/provider/oracle.py

```diff
@@ -1,6 +1,7 @@
 # =================================================================
 #
 # Authors: Andreas Kosubek <andreas.kosubek@ama.gv.at>
+# Authors: Moritz Langer <moritz.b.langer@gmail.com>
 #
 # Copyright (c) 2023 Andreas Kosubek
```

Comment:
Can you also add yourself to the copyright?

Reply:
Changed accordingly